app/api_alpha/views.py

```diff
@@ -1954,7 +1956,9 @@ def get_stats(request):
         "building_counts": bdg_count_kpi.value,
         "api_calls_since_2024_count": api_calls_since_2024_count,
         "contributions_count": contributions_count,
```

Comment:
Pour faire la distinction signalement/édition. Ca nécessite de faire une retouche sur le front également. Je l'ai préparée ici : https://github.com/fab-geocommuns/RNB-site/pull/294

```suggestion
        "reports_count": contributions_count,
```